src/effects.cpp

```diff
@@ -447,20 +447,37 @@ void LoadEffectFactories()
         ADD_EFFECT(EFFECT_STRIP_METEOR, MeteorEffect, 1, 1, 5, .15, .25);
         ADD_EFFECT(EFFECT_STRIP_METEOR, MeteorEffect); // Rainbow palette
 
+    #elif PLATECOVER
+
+        #ifndef EFFECT_SET_VERSION
+            #define EFFECT_SET_VERSION  3
+        #endif    

```

Comment:
I'm changing this to 1 for PLATECOVER; the effect set version is project specific.

```suggestion
        #ifndef EFFECT_SET_VERSION
            #define EFFECT_SET_VERSION  1
        #endif    
```